### PR TITLE
#333 telephone field on user profile form

### PR DESCRIPTION
**Pull request by filipgorczynski**

### CODE REVIEW

File: apps/volontulo/forms.py [hunk 1]
```diff
@@ -40,18 +50,20 @@ def is_valid(self):
...
 
         current_password = self.cleaned_data['current_password']
+        new_password = self.cleaned_data['new_password']
+        confirm_new_password = self.cleaned_data['confirm_new_password']
         user = User.objects.get(id=self.cleaned_data['user'])
 
-        if not user.check_password(current_password):
-            raise ValidationError(u"Aktualne hasło jest błędne")
+        if (
+                current_password and
```
Comment (Yog) [inline apps/volontulo/forms.py:58]: too big tab indent?

Comment (filipgorczynski) [inline apps/volontulo/forms.py:58]: pep8 require 2 indents here.

File: apps/volontulo/forms.py [hunk 2]
```diff
@@ -29,9 +29,19 @@ class Meta(object):
...
+        widget=forms.PasswordInput(),
+        required=False
+    )
+    confirm_new_password = forms.CharField(
+        widget=forms.PasswordInput(),
+        required=False
+    )
     user = forms.CharField(widget=forms.HiddenInput())
 
     def is_valid(self):
```
Comment (radoslawroszkowiak) [inline apps/volontulo/forms.py:47]: Do we test this method somehow? I think testing the validation is necessary.

**Comment by Yog** [comment]
I gues all that is necessary here are the validation tests @radoslawroszkowiak pointed out?
And then we can merge away?

**Pull request merged**